transformer.js

```diff
@@ -0,0 +1,67 @@
+const fs = require('fs').promises;
+
+const transformer = async(dir) => {
+    return fs.readFile(dir, 'utf-8')
+    .then(data => {
+        console.log(data);
+        return data;
+    })
+    .then(data => {
+        data = lowerCase(data);
+        return data;
+    })
+    .then(data => {
+        data = upperCase(data);
+        return data;
+    })
+    .then(data => {
+        data = reverseString(data);
+        return data;
+    })
+    .then(data => {
+        data = writer(dir, data);
+        return data;
+    })
+    .catch(err => {
+        console.log(err);
+        return err;
+    });
+}
+
+const lowerCase = (data) => {
```

Comment:
only critique is that I believe this lab was to have you remove the capital letters completely, not just lowercase them.  However, since this was more about promises in this lab, no points were taken off. Just might be some good practice if you have time in the future